imgcreate/creator.py

```diff
@@ -1,9 +1,10 @@
 #
 # creator.py : ImageCreator and LoopImageCreator base classes
 #
-# Copyright 2007, Red Hat  Inc.
+# Copyright 2007, Red Hat, Inc.
 # Copyright 2016, Kevin Kofler
 # Copyright 2016, Neal Gompa
+# Copyright 2017, Fedora Project
```

Comment:
This is wrong unless you intend to assign rights to the Fedora Project... I think you meant Sugar Labs?